plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar310/TreeBuilder310.java

```diff
@@ -679,13 +680,17 @@ public final SimpleNode onCloseNode(SimpleNode n, int arity) throws Exception {
 
             case JJTATTR:
                 if (arity == 1) {
+                    if (isPeekedNodeWildcardPattern()) {
+                        stack.popNode();
+                        return getWildcardPattern();
+                    }
                     return stack.popNode();
                 } else if (arity >= 2) {
                     SimpleNode peekedNode = stack.peekNode();
                     if (peekedNode instanceof patternType) {
                         popSurplus(arity, 2);
                         patternType pattern = (patternType) securePop(patternType.class);
-                        Name arg = makeName(name_contextType.KeywordName);
+                        Name arg = makeName(Name.AugLoad);
```

Comment:
What's the use-case here?

AugLoad is usually something as:

`x += 1`

(so, maybe a simple `Load` would be better here?)

Reply:
We could use Name.Artificial here? Because we should somehow handle later this name and properly ignore the keyword arg in the scope

Reply:
Yes, that's ok.